src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -2197,6 +2197,10 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
 
             bool atleastOneExactFilePathWasLookedAtAndNotFound = false;
 
+            var allProjects = new List<ProjectRootElement>();
+            bool containsWildcards = EscapingUtilities.ContainsEscapedWildcards(importElement.Project) ||
+                                     FileMatcher.HasWildcards(importElement.Project);
```

Comment:
What's the motivation for checking for escaped wildcards? Shouldn't only real wildcards matter?